psalm-baseline.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="3.9.3@2e4154d76e24d1b4e59e6cc2bebef7790cb9e550">
+  <file src="src/Roave/SecurityAdvisories/Advisory.php">
+    <MixedReturnTypeCoercion occurrences="2">
+      <code>$versionConstraints</code>
+      <code>list&lt;VersionConstraint&gt;</code>
+    </MixedReturnTypeCoercion>
+  </file>
+  <file src="src/Roave/SecurityAdvisories/AdvisorySources/GetAdvisoriesFromFriendsOfPhp.php">
+    <MixedArgument occurrences="1">
+      <code>Yaml::parse(file_get_contents($filePath), Yaml::PARSE_EXCEPTION_ON_INVALID_TYPE)</code>
+    </MixedArgument>
+    <MixedReturnTypeCoercion occurrences="2">
+      <code>SplFileInfo[]</code>
+    </MixedReturnTypeCoercion>
+  </file>
+  <file src="src/Roave/SecurityAdvisories/Component.php">
+    <MixedReturnTypeCoercion occurrences="2">
+      <code>$constraints</code>
```

Comment:
`usort()` is using by-ref here: maybe we can find a way to have non-by-ref sort?

Reply:
Well, if you want to change the code, The easiest way to solve is to use native usort and check for false?

Reply:
I'm just wondering if a solution is feasible, or if we should just give up and add a suppression...

Reply:
The issue is not really the by-ref though. Even a non-by-ref function overrided by codingmachine/safe without generics notation will return mixed...